_data-prepper/pipelines/configuration/processors/anomaly-detector.md

```diff
@@ -0,0 +1,86 @@
+---
+layout: default
+title: Anomaly detector
+parent: Processors
+grand_parent: Pipelines
+nav_order: 45
+---
+
+# Anomaly detector
+
+## Overview
+
+The `Anomaly detector` processor takes structured data and runs anomaly detection algorithms on fields you can configure in the data. The data must be either an integer or real number in order for the the anomaly detection algorithm to detect anomalies. We recommend that you deploy the `Aggregate` processor in a pipeline before the `Anomaly detector` processor to achieve the best results.  This is because the `Aggregate` processor aggregates events with same keys onto the same host.
+
+This processor uses the random cut forest mode to detect anomalies. 
```

Comment:
Are there other modes? If so, what are they? They will need as much information as the random_cut_forest mode has.

Reply:
Currently, there is only one mode.